lib/control-app.js

```diff
@@ -136,5 +136,14 @@ module.exports = function(config) {
         });
     });
 
-    return app;
```

Comment:
The return value isn't used anywhere?

Reply:
It used to be here (on current master)

https://github.com/airtap/airtap/blob/86e5d916534505bd177ad7a9c80f810738c1666f/lib/airtap.js#L60-L63

Reply:
But removed that method and the code inside that is now inside `control-app.js`, which simply calls back with the port and close function.